raresGabriel01/home-assignment
# [ISSUE-29] removed t-progress errors

# Issue link
#29 
# Description
Removed t-progress error messages and updated the unit tests.
Also added a `OnPush` change detection strategy.
# How to test?
Open the storybook on the last 3 tabs and ensure no errors are thrown. Ensure the radius/progress default to the min/max values instead.
# Attachments
